apps/crn-server/src/controllers/reminders.ts

```diff
@@ -50,6 +50,30 @@ export default class Reminders implements ReminderController {
           };
         }
 
+        if (
+          reminder.entity === 'Research Output' &&
+          reminder.type === 'Draft'
+        ) {
+          if (reminder.data.associationType === 'team') {
+            return {
+              id: reminder.id,
+              entity: reminder.entity,
+              href: sharedResearch({}).researchOutput({
+                researchOutputId: reminder.data.researchOutputId,
+              }).$,
+              description: `**${reminder.data.createdBy}** on **${reminder.data.associationName}** created a draft output ${reminder.data.title}.`,
```

Comment:
there's a colon missing
```suggestion
              description: `**${reminder.data.createdBy}** on **${reminder.data.associationName}** created a draft output: ${reminder.data.title}.`,
```